packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -226,7 +225,28 @@ class ScaleGestureRecognizer extends OneSequenceGestureRecognizer {
   ScaleGestureRecognizer({
     Object? debugOwner,
     PointerDeviceKind? kind,
-  }) : super(debugOwner: debugOwner, kind: kind);
+    this.dragStartBehavior = DragStartBehavior.down,
+  }) : assert(dragStartBehavior != null),
+    super(debugOwner: debugOwner, kind: kind);
+
+  /// Determines at what point the gesture is calculated from.
```

Comment:
nit: a bit confusing to read but I'm not sure how best to put this. "considered started and used as the starting point in all calculations involving this gesture"?

Reply:
Here's what I changed it to, let me know if you think of something better: "Determines what point is used as the starting point in all calculations involving this gesture."

I specifically didn't want to imply that the time that onStart will be called is affected by this.  In my experimentation with onHorizontalDragStart, which already uses DragStartBehavior, its onStart is not affected, so I made sure this one isn't affected either.

Reply:
I also added to the onStart docs just below here to make this more clear.